app/models/bulk_download.rb

```diff
@@ -25,4 +30,161 @@ def convert_params_to_json
   def self.viewable(user)
     user.bulk_downloads
   end
+
+  def validate_access_token(access_token)
+    return self.access_token == access_token
```

Comment:
could you also do `self.access_token.present?` here? I'm wondering if we'll ever get into a situation where access_token is "" or accidentally not set, and then anyone can access it